sherlockml/clients/report.py

```diff
@@ -0,0 +1,174 @@
+# Copyright 2018 ASI Data Science
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from collections import namedtuple
+
+from marshmallow import Schema, fields, post_load, pre_load
+
+from sherlockml.clients.base import BaseClient
+
+Report = namedtuple(
+    "Report", ["created_at", "name", "id", "description", "active_version"]
+)
+
+VersionedReport = namedtuple(
```

Comment:
Can we call this (and the associated schema) something like 'ReportWithVersions'? All reports are 'versioned' so I think this name may be slightly misleading.